src/Tools/ReleaseNotesUtil/RuleFileContent.cs

```diff
@@ -9,6 +9,8 @@ namespace ReleaseNotesUtil
     internal class RuleFileContent
     {
         [DataMember]
+#pragma warning disable CA1002 // Do not expose generic lists
```

Comment:
CA1002  does not seem to be a build warning. We normally avoid adding source suppressions for IDE-suggestions.